keys/keys.js

```diff
@@ -1,2 +1,4 @@
 module.exports = {
+serverConnectionUrl: 'mongodb+srv://jagrti:JHA8875091601@cluster0-ul1ff.mongodb.net/login_details?retryWrites=true&w=majority',
```

Comment:
key should be "URI" and not "serverConnectionUrl", else it'll throw an error in the backend logic